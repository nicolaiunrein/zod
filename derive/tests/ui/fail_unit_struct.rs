use zod_derive::Zod;

struct Ns;
impl zod_core::Namespace for Ns {
    const NAME: &'static str = "Custom_Ns";
}

#[derive(Zod)]
#[zod(namespace = "Ns")]
struct X;

fn main() {}
