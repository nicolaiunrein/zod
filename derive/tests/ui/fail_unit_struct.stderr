error: Unsupported shape `no fields`. Expected named fields or unnamed fields.
 --> tests/ui/fail_unit_struct.rs:8:10
  |
8 | #[derive(Zod)]
  |          ^^^
  |
  = note: this error originates in the derive macro `Zod` (in Nightly builds, run with -Z macro-backtrace for more info)
